classes/class-wc-rest-connect-address-normalization-controller.php

```diff
@@ -96,11 +105,29 @@ public function update_items( $request ) {
 		$response->normalized->name = $name;
 		$response->normalized->company = $company;
 		$response->normalized->phone = $phone;
+		$is_trivial_normalization = isset( $response->is_trivial_normalization ) ? $response->is_trivial_normalization : false;
+
+		if ( $is_trivial_normalization ) {
+			$this->update_address( $order_id, $type, $response->normalized );
+		}
 
 		return array(
 			'success' => true,
 			'normalized' => $response->normalized,
-			'is_trivial_normalization' => isset( $response->is_trivial_normalization ) ? $response->is_trivial_normalization : false,
+			'is_trivial_normalization' => $is_trivial_normalization,
+		);
+	}
+
+	public function accept_suggestion( $request ) {
+		$data     = $request->get_json_params();
+		$order_id = $data[ 'orderId' ];
+		$type     = $data[ 'type' ];
+		$address  = $data[ 'address' ];
+
+		$this->update_address( $order_id, $type, $address );
```

Comment:
I was referring to the `Rates` request on the Purchase Label dialog. The point is, use one of the endpoints that are already there and are hit in the course of a label purchase, instead of creating a new one.